### ISSUE TITLE
Add link to the lemmy community as well

### ISSUE THREAD

**Issue by db0** [RikudouSage/AiHordeFrontpage#4]
https://lemmy.dbzer0.com/c/aihorde

**Comment by RikudouSage** [comment]
I added it as a lemmyverse.link link, is that fine by you?

**Comment by db0** [comment]
hmm, won't that make it more confusing for newbies to lemmy though? Asking them to choose and all? lemmyverse.link is great for lemmy users, but for outsiders is adding an extra level of choice about something they don't know about. 

That gives me an idea. Can the lemmyverse.link get like a query arg which automatically moves on to the pointed instance after 3 seconds?

**Comment by RikudouSage** [comment]
I thought the "Just take me there!" button would be enough? I could implement such a parameter though I'm not sure it's a good UX, 3 seconds is not enough to read the text if you want to.

**Comment by db0** [comment]
I forgot that button exists tbh :D

Hmm, I still think for someone new, clicking a link and then being asked to make a choice about something you don't understand might just make them close the browser window.

Make the parameter. When it's invoked, put the "Just take me there" button in the front and highlight it a bit more. Add a 5-10 seconds timer as well and say "If you don't know what to do, just press the 'just take me there' and we'll also automatically redirect you to it in 5/10 seconds". something like that

Basically something to avoid overwhelming newbs, yanno?

**Comment by RikudouSage** [comment]
For now I've changed it to a direct link, I'll make a new PR with lemmyverse.link when I figure out how exactly I want to do that.
